src/abstractdataframe/join.jl

```diff
@@ -280,7 +311,9 @@ join(name, job2, on = :ID => :identifier)
 function Base.join(df1::AbstractDataFrame,
                    df2::AbstractDataFrame;
                    on::Union{<:OnType, AbstractVector{<:OnType}} = Symbol[],
-                   kind::Symbol = :inner, makeunique::Bool=false)
+                   kind::Symbol = :inner, makeunique::Bool=false,
+                   check_uniqueness::Tuple{Bool,Bool}=(false, false))
```

Comment:
Would be nice to also allow `Pair{Bool,Bool}` as we do for `on`.

Also don't add space below.